lib/TerrainHandler.cpp

```diff
@@ -0,0 +1,316 @@
+/*
+ * Terrain.cpp, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+
+#include "StdInc.h"
+#include "TerrainHandler.h"
+#include "VCMI_Lib.h"
+#include "CModHandler.h"
+#include "CGeneralTextHandler.h"
+
+VCMI_LIB_NAMESPACE_BEGIN
+
+//regular expression to change id for string at config
```

Comment:
Is this comment needed now?

Reply:
Not exactly sure what was its purpose even before my changes. I guess I'll remove it then.